app/templates/signatures/ongoing.hbs

```diff
@@ -1,104 +1,120 @@
+                        @accessLevel={{piece.accessLevel}}
+                        @isEditable={{false}}
+                      />
+                    {{/if}}
+                  {{/if}}
+                </td>
+                <td>
+                  <AuPill @size="small">
+                    {{#let piece.documentContainer.type as |type|}}
+                      {{#if type}}
+                        {{type.label}}
+                      {{else}}
+                        {{t "no-document-type"}}
+                      {{/if}}
+                    {{/let}}
+                  </AuPill>
+                </td>
+              {{/let}}
+              <td>
+                {{#let (await (this.getMandateeNames signFlow)) as |names|}}
+                  {{#if names}}
+                    {{join ", " names}}
+                  {{else}}
+                    -
+                  {{/if}}
+                {{/let}}
+              </td>
+              <td>
+                {{date signFlow.decisionActivity.startDate}}
+              </td>
+              <td>
+                <SignaturePill
+                  @piece={{signMarkingActivity.piece}}
+                  @signMarkingActivity={{signMarkingActivity}}
+                />
+              </td>
             {{/let}}
-          </AuPill>
-        </td>
-      {{/let}}
-      <td>
-        {{#let (await (this.getMandateeNames signFlow)) as |names|}}
-          {{#if names}}
-            {{join ", " names}}
-          {{else}}
-            -
-          {{/if}}
-        {{/let}}
-      </td>
-      <td>
-        {{date signFlow.decisionActivity.startDate}}
-      </td>
-      <td>
-        <SignaturePill
-          @piece={{signMarkingActivity.piece}}
-          @signMarkingActivity={{signMarkingActivity}}
-        />
-      </td>
-      <td>
-        {{#let (await (this.getSigningHubUrl signFlow)) as |url|}}
-          {{#if url}}
-            <AuLinkExternal
-              @skin="button-naked"
-              @icon="chevron-right"
-              @hideText={{true}}
-              href={{url}}
-            >
-              {{t "open"}}
-            </AuLinkExternal>
-          {{else}}
-            <AuButton
-              @skin="naked"
-              @icon="link-broken"
-              @hideText={{true}}
-              @disabled={{true}}
-            >
-              {{t "open"}}
-            </AuButton>
-          {{/if}}
-        {{/let}}
-      </td>
-    {{/let}}
-  </:body>
-</KDataTable>
+              <td class="auk-u-text-align--right">
+                <AuButton
+                  @skin="naked"
+                  @icon="chevron-right"
+                  @hideText={{true}}
+                >
+                  {{t "open"}}
+                </AuButton>
+              </td>
+                </:body>
+              </KDataTable>
+            {{else}}
+              <Auk::EmptyState
+                @bordered={{true}}
+                @message={{t "no-results-found-modify-term"}}
```

Comment:
This one needs to change, there is no search term here

Reply:
Maybe also differentiate between no items because of filters and no items because there are no items in the database. So 2 different messages

Reply:
Changed this to use the built-in KDataTable message (more generic "Geen resultaten gevonden"). No way to know if this is because of filters or not